src/custom/pages/Faq/index.tsx

```diff
@@ -758,17 +758,21 @@ export default function Faq() {
 
           <p>
             The affiliate program will initially run as a trial, and therefore there are currently no announced rewards,
-            the tracking however, has already started.
+            the tracing however, has already started.
```

Comment:
Why are we calling this `tracing`?

@avsavsavs @anxolin

Reply:
No idea! I think is an error and i didn't address your comment 😅

Reply:
https://github.com/gnosis/cowswap/pull/1926